### PR TITLE
Html comments

### PR DESCRIPTION
**Pull request by rizwans-cuelogic**
resolved html comments

### CODE REVIEW

File: project4/project/ebs/static/ebs/css/common.css [hunk 1]
```diff
@@ -0,0 +1,118 @@
+#footer{
+			height:430px;
```
Comment (sagarp-cuelogic) [inline project4/project/ebs/static/ebs/css/common.css]: why footer needs fixed height?

File: project4/project/ebs/static/ebs/css/common.css [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+}
+.white-text{
+	color:white;
+}
+
+.marginlogo{
+	margin-left:-27px;
+
+}
+.white-text{
```
Comment (sagarp-cuelogic) [inline project4/project/ebs/static/ebs/css/common.css]: duplicate css

File: project4/project/ebs/static/ebs/css/home.css [hunk 3]
```diff
@@ -1,383 +1,206 @@
...
-	font-family:Verdana,Geneva,sans-serif;
-	line-height:21px;
-
+body{
+	font-family:'Open Sans',arial,sans-serif;
+	
 }
-
-.fontfamily2{
+h1, h2, h2,h3,h4{
```
Comment (sagarp-cuelogic) [inline project4/project/ebs/static/ebs/css/home.css]: remove duplicate code

File: project4/project/ebs/static/ebs/css/home.css [hunk 4]
```diff
@@ -1,383 +1,206 @@
...
-
-}
-
-.hrcol{
-	border-top:2px solid #009688;
-
-}
-
-.hrcol1{
+.latest-hr{
```
Comment (sagarp-cuelogic) [inline project4/project/ebs/static/ebs/css/home.css]: avoid unnecessary use of hr element, you can use border for the same

File: project4/project/ebs/static/ebs/css/common.css [hunk 5]
```diff
@@ -0,0 +1,118 @@
...
+	vertical-align: center;
+	color:white;
+	text-decoration:none;
+}
+
+.copyright{
+	color:white;
+	vertical-align: center;
+	line-height: 40px;
+	padding-left:14%;
```
Comment (sagarp-cuelogic) [inline project4/project/ebs/static/ebs/css/common.css]: Remove padding.

File: project4/project/ebs/static/ebs/css/common.css [hunk 6]
```diff
@@ -0,0 +1,118 @@
...
+
+.footer2-padding li a{
+	vertical-align: center;
+	color:white;
+	text-decoration:none;
+}
+
+.copyright{
+	color:white;
+	vertical-align: center;
```
Comment (sagarp-cuelogic) [inline project4/project/ebs/static/ebs/css/common.css]: wrong CSS propery

File: project4/project/ebs/static/ebs/css/common.css [hunk 7]
```diff
@@ -0,0 +1,118 @@
...
+	line-height: 40px;
+	padding-left:14%;
+	
+}
+.white-text{
```
Comment (sagarp-cuelogic) [inline project4/project/ebs/static/ebs/css/common.css]: Avoid presentation or location specific words in the name
Study provided reference links for CSS naming conventions

File: project4/project/ebs/static/ebs/css/home.css [hunk 8]
```diff
@@ -1,383 +1,206 @@
...
 	font-family: 'Roboto',sans-serif;
-	line-height:25px;
 }
 
-.fontfamily3{
-	font-family:'Open Sans',arial,sans-serif;
-	font-size:14px;
-	line-height: 30px;
-}
+.font-verdana{
```
Comment (sagarp-cuelogic) [inline project4/project/ebs/static/ebs/css/home.css]: Avoid presentation or location specific words in the name

File: project4/project/ebs/static/ebs/css/home.css [hunk 9]
```diff
@@ -1,383 +1,206 @@
...
 h2{
 	font-size:25px;
 }
 
-white-text{
-	color:white;
-}
-
 a{
 	text-decoration: none !important;
```
Comment (sagarp-cuelogic) [inline project4/project/ebs/static/ebs/css/home.css]: remove !important

File: project4/project/ebs/templates/ebs/home.html [hunk 10]
```diff
@@ -105,17 +96,20 @@
...
+						<img  class="banner1-imgbig img-responsive " src="{% static 'ebs/img/pro1.jpg' %} " /> 
+						</a>
 	
 					</div>
-					<div id="text1"><div class="text-block"><span class=" white-text  font1"><a href="#"><p class="banner-line">PHOTOGRAPHY</p></a></span></div>
-					<a href="#"><h3> Celebrity Make-up artist gary cockrill shows you beaty tips</h3></a>
-					<a href="#"><span class="white-text marginl font-author"><b>David Lee</b></span><span class="white-text font-author">  -  September 24,    2016</span></a> 
+					<div id="banner1-text">
+					<a href="#"><span class="catagory-block left-margin">PHOTOGRAPHY</span></a>
+					<a href="#"><h3>Celebrity Make-up artist gary cockrill shows you beaty tips</h3></a>
```
Comment (sagarp-cuelogic) [inline project4/project/ebs/templates/ebs/home.html]: inline element should not wrap block or inline-block element

File: project4/project/ebs/templates/ebs/home.html [hunk 11]
```diff
@@ -105,17 +96,20 @@
...
+					<a href="#"><span class="white-text left-margin author-font"><b>David Lee</b></span><span class="white-text font-author">  -  September 24,    2016</span></a>
```
Comment (sagarp-cuelogic) [inline project4/project/ebs/templates/ebs/home.html]: avoid using multiple classes for single purpose, it will help you to reduce code.

**Review by sagarp-cuelogic — COMMENTED** [review]
- For repeated layout you can use same class, avoid using muliple classes with same css properties.
 - Giving fixed height and width to the element is wrong practice. Remove fixed height and width from the CSS.
 - Code should be always neat and clean. remove extra spaces and blank lines, follow the same tab format while coding.
 - Top bar navigation and footer bottom are vertical not aligned.
 - It is not required to give classnames to every element. you can use ".parent-div-class h1 / p / li{CSS propery}"

File: project4/project/ebs/static/ebs/css/home.css [hunk 12]
```diff
@@ -1,383 +1,216 @@
...
-	line-height: 1px;
-}
-.font-weigh{
-	font-weight:bold;
+	
 }
-
-p{
-	margin:0px;
+h1, h2, h2,h3,h4{
```
Comment (sagarp-cuelogic) [inline project4/project/ebs/static/ebs/css/home.css]: duplicate code is not yet removed.

File: project4/project/ebs/static/ebs/css/home.css [hunk 13]
```diff
@@ -1,383 +1,216 @@
...
-	margin-top:10px;
+.banner-text .label-default:hover{
+	background-color:#009688;
 }
 
-.margin3{
-	margin-top:20px;
+.banner-text span{
+	font-size:11px;
+	line-height: 1px;
```
Comment (sagarp-cuelogic) [inline project4/project/ebs/static/ebs/css/home.css]: Line-height 1px is not making any difference in the appearance. so remove it.

File: project4/project/ebs/static/ebs/css/home.css [hunk 14]
```diff
@@ -1,383 +1,216 @@
...
 
-hr{
-	margin-top:10px;
-}
-h2{
-	font-size:25px;
+.latest-blog{
+	position:relative;
+	height:285px;
+	width:90%;
```
Comment (sagarp-cuelogic) [inline project4/project/ebs/static/ebs/css/home.css]: Fix width to the div is not required, when you are using bootstrap framework.

File: project4/project/ebs/templates/ebs/home.html [hunk 15]
```diff
@@ -98,222 +89,229 @@
...
-									<div class="text-bannerside">
-										<a href="#"><div class="text-block2"><p class="font1">TECH</p></div>
-										<h4 class="fontfamily1">Gigabyte GeForce GTX 950 Review: Pricing Is Everything</h4>
-										</a>
-									</div>
-								
-								</div>
+				<div  class="col-md-6">
+					<div class="row">
+						<div class="col-md-6 screen1">
```
Comment (sagarp-cuelogic) [inline project4/project/ebs/templates/ebs/home.html]: unwanted class names are still there, please remove them. - screen1

File: project4/project/ebs/templates/ebs/home.html [hunk 16]
```diff
@@ -98,222 +89,229 @@
...
-					</div>
-					<div id="text1"><div class="text-block"><span class=" white-text  font1"><a href="#"><p class="banner-line">PHOTOGRAPHY</p></a></span></div>
-					<a href="#"><h3> Celebrity Make-up artist gary cockrill shows you beaty tips</h3></a>
-					<a href="#"><span class="white-text marginl font-author"><b>David Lee</b></span><span class="white-text font-author">  -  September 24,    2016</span></a> 
+					<div class="banner-blog">
+						<a  href="#"><img class="large" src="{% static 'ebs/img/pro1.jpg' %}"/></a>
+						<div class="banner-text">
+							<a href="#" class="label label-default">photography</a>
+							<h3><a href="#">Celebrity Make-up artist gary cockrill shows you beaty tips</a></h3>
+							<a href="#"><span><strong>David Lee</strong></span></a>
```
Comment (sagarp-cuelogic) [inline project4/project/ebs/templates/ebs/home.html]: no need to use multiple elements, remove extra spans as suggested.

File: project4/project/ebs/templates/ebs/home.html (hunk 15 above)
Comment (rizwans-cuelogic) [reply project4/project/ebs/templates/ebs/home.html]: that is not unwanted i need it to leave margin in media queries

File: project4/project/ebs/templates/ebs/home.html [hunk 17]
```diff
@@ -98,222 +89,229 @@
...
+								<span class="latest-author">   -   September 24,   2016</span>
+								<a href="#"><span class="pull-right view-block">0</span></a>
+
 							</div>
+							</div>	
 						</div>
 					</div>
 					
-					<div class="row">
+					<div class="row " >
```
Comment (sagarp-cuelogic) [inline project4/project/ebs/templates/ebs/home.html]: remove unwanted extra spaces and blank lines.

File: project4/project/ebs/templates/ebs/home.html [hunk 18]
```diff
@@ -98,222 +89,229 @@
...
-								<a  href="#"><img  class="img-responsive latest-img" src="{% static 'ebs/img/por1.jpg' %} "/></a>
-								<div class="latest-text">
-									<a href="#"><div class="text-block1"><p class="white-text font1">HEALTH</p></div></a>
-									<a class="latest-color fontfamily2" href="#"><h4 class="fontsize1">The workout That Burns More Calories Than Running</h4></a>
-									<span class="font-author"><a href="#"><b>David Lee</b></a></span><span class="font-author">    -   September 24,   2016</span><a href="#"><span class="pull-right view-block">0</span></a> 
-								</div>
+							<a href="#"><img  src="{% static 'ebs/img/por1.jpg' %} "/></a>
+
+							<div class="latest-text">
+								<a href="#"><span class="label label-default">Health</span></a>
```
Comment (sagarp-cuelogic) [inline project4/project/ebs/templates/ebs/home.html]: apply the classes to "a" tag and remove the "span"

File: project4/project/ebs/templates/ebs/home.html [hunk 19]
```diff
@@ -322,116 +320,162 @@ <h4 class="fontfamily1">Fashion Week Parties: Night 4</h4></a>
 				<div class="col-md-4 ">
 					<!-- MOST POPULAR -->
 
-					<h4><strong>MOST POPULAR</strong></h4>
-					<hr class="hrcol">
+					<h4 class="popular-hr"><strong>MOST POPULAR</strong></h4>
```
Comment (sagarp-cuelogic) [inline project4/project/ebs/templates/ebs/home.html]: use different heading tags to show different titles. e.g. h1, h2, h3, h4, h5, h6

File: project4/project/ebs/templates/ebs/home.html [hunk 20]
```diff
@@ -322,116 +320,162 @@ <h4 class="fontfamily1">Fashion Week Parties: Night 4</h4></a>
...
-							<div class="text-block3 inlineb"><a href="#"><p class="white-text font1">PHOTOGRAPHY</p></a></div><span class="font-author">   -   September 24,   2016</span>
+							<h4 ><a  href="#"> Celebrity Make-up artist gary cockrill shows you beaty tips</a></h4>
+							
+							<a href="#"><span class="label label-default">PHOTOGRAPHY</span></a>
+							<span class="popular-author">   -   September 24,   2016</span>
+						</div>
 						</div>
 					</div>	
-					<hr class="margin3">
+					<hr class="top-margin-20">
```
Comment (sagarp-cuelogic) [inline project4/project/ebs/templates/ebs/home.html]: remove the class applied to hr, hr has default margin of 20px in bootstrap.

File: project4/project/ebs/templates/ebs/home.html [hunk 21]
```diff
@@ -322,116 +320,162 @@ <h4 class="fontfamily1">Fashion Week Parties: Night 4</h4></a>
...
 						<div class="row">
 							<div class="col-md-12">
-								<a  class="latest-color" href="#">
-								<h4 class="fontsize3">Georgian soldier helped in attack near US base in Afghanistan</h4></a>
-								<div class="text-block2 inlineb"><a href="#"><p class="white-text font1">ARTS</p></a></div> &nbsp;<span class="font-author"><a href="#" class="author-color"><b>David Lee</b></a></span><span class="font-author">    -    September 24,   2016</span></a><a href="#"><span class="pull-right view-block">0</span></a>
+								<div class="must-blog">
+								<h4><a  href="#">Georgian soldier helped in attack near US base in Afghanistan</a></h4>
+							
+								<a href="#"><span class="label label-default">ARMY</span></a>
+								 &nbsp;
```
Comment (sagarp-cuelogic) [inline project4/project/ebs/templates/ebs/home.html]: remove the blank lines and unwanted code.

File: project4/project/ebs/templates/ebs/home.html [hunk 22]
```diff
@@ -322,116 +320,162 @@ <h4 class="fontfamily1">Fashion Week Parties: Night 4</h4></a>
 				<div class="col-md-4 ">
```
Comment (sagarp-cuelogic) [inline project4/project/ebs/templates/ebs/home.html]: HTML5 semantic markup not used. use section, main, aside and other semantic markup while using HTML5

File: project4/project/ebs/templates/ebs/home.html [hunk 23]
```diff
@@ -502,13 +544,13 @@ <h4 class="fontsize3">Georgian soldier helped in attack near US base in Afghanis
...
 			</div>
-			<div class="modal-body">
+			<div class="modal-body modal-padding">
 			<form id="signinform" method="post" data-toggle="validator">
 				<div id="message">  </div>
 				{% csrf_token %}
 				<br><div class="form-group ">
 						
-						<label class="control-label">Username</label>&nbsp;&nbsp;&nbsp;<span style="color:red">{{loginform.username.errors}}</span>
+						<label class="control-label">Username</label><span style="color:red">{{loginform.username.errors}}</span>
```
Comment (sagarp-cuelogic) [inline project4/project/ebs/templates/ebs/home.html]: remove inline styles.

File: project4/project/ebs/templates/ebs/home.html [hunk 24]
```diff
@@ -502,13 +544,13 @@ <h4 class="fontsize3">Georgian soldier helped in attack near US base in Afghanis
 				<button class="pull-right btn2" type="button" data-dismiss="modal" aria-hidden="true">&times;</button>
 				<h2 class="text-center">Sign in</h2>
 			</div>
-			<div class="modal-body">
+			<div class="modal-body modal-padding">
 			<form id="signinform" method="post" data-toggle="validator">
 				<div id="message">  </div>
 				{% csrf_token %}
 				<br><div class="form-group ">
```
Comment (sagarp-cuelogic) [inline project4/project/ebs/templates/ebs/home.html]: remove unwanted br

File: project4/project/ebs/static/ebs/css/home.css [hunk 25]
```diff
@@ -1,383 +1,199 @@
...
+	margin-top:50px;
 }
-
-.footer2-padding li a{
-	vertical-align: center;
-	color:white;
-	text-decoration:none;
+.latest-hr{
+	border-bottom: 2px solid #009688;
+	margin-left:-1px;
```
Comment (sagarp-cuelogic) [inline project4/project/ebs/static/ebs/css/home.css]: Remove margin-left as it is not required.

File: project4/project/ebs/templates/ebs/home.html [hunk 26]
```diff
@@ -98,354 +84,339 @@
...
 	<!--Latest blogs and must read -->
-	<div class="contain1">
+	<div class="top-margin-40">
 		<div class="container">
 			<div class="row">
-				<div class="col-md-8">
-					<h4><strong>LATEST BLOGS</strong></h4>
-					<hr class="hrcol1">
-					<div class="row">
+				<section class="col-md-8">
```
Comment (sagarp-cuelogic) [inline project4/project/ebs/templates/ebs/home.html]: use main tag here

File: project4/project/ebs/templates/ebs/home.html [hunk 27]
```diff
@@ -98,354 +84,339 @@
...
-				
+				</div>
 			</div>
 		</div>			
-	</div>	
-
+	</section>	
 	<!--Latest blogs and must read -->
-	<div class="contain1">
+	<div class="top-margin-40">
```
Comment (sagarp-cuelogic) [inline project4/project/ebs/templates/ebs/home.html]: use section tag here

File: project4/project/ebs/templates/ebs/home.html [hunk 28]
```diff
@@ -98,354 +84,339 @@
...
 							<div class="col-md-12">
-								<a class="latest-color" href="#">
-								<h4 class="fontsize3">Gigabyte GeForce GTX 950 Review: Pricing Is Everything</h4></a>
-								<div class="text-block2 inlineb"><a href="#"><p class="white-text font1">TECH</p></a></div>
-							 &nbsp;<span class="font-author"><a href="#" class="author-color"><b>David Lee</b></a></span><span class="font-author">    -   September 24,   2016</span></a><a href="#"><span class="pull-right view-block">0</span></a>
-
+								<div class="must-blog">
+								<h5><a  href="#">Gigabyte GeForce GTX 950 Review: Pricing Is Everything</a></h5>
+								<a  href="#" class="label label-default">TECH</a>
+								 &nbsp;<span class="must-author"><a href="#"><strong>David Lee</strong></a>  -   September 24,   2016</span>
```
Comment (sagarp-cuelogic) [inline project4/project/ebs/templates/ebs/home.html]: Remove &nbsp; and use margin for spacing between two elements.

File: project4/project/ebs/templates/ebs/home.html [hunk 29]
```diff
@@ -98,354 +84,339 @@
...
+
 					</div>
 					<!-- Catogories -->
-					<div class="margin1">
-						<h4><strong>CATAGORIES</strong></h4>
-						<hr class="hrcol">
+					<div class="top-margin-30">
+						<h4 class="latest-hr"><strong>CATAGORIES</strong></h4>
+						
 						<div class="row">
```
Comment (sagarp-cuelogic) [inline project4/project/ebs/templates/ebs/home.html]: use simple ul li structure for display simple list items

**Pull request merged**